ajax/libs/emojionearea/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "emojionearea", 
+  "description": "EmojiOne Area is a small jQuery plugin that allows you to transform any html element into simple WYSIWYG editor with ability to use Emojione icons. The end result is a secure text/plain in which the image icons will be replaced with their Unicode analogues.",
+  "filename": "emojionearea.min.js",
+  "keywords": [
+	"emoji", 
+	"emojione", 
+	"picker", 
+	"emojionearea"
+  ],
+  "author": "Andrey Izman <izmanw@gmail.com>",
+  "homepage": "http://mervick.github.io/emojionearea/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mervick/emojionearea.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mervick/emojionearea.git",	
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "emojionearea*.+(css|map|js)"
```

Comment:
Let's use `**/*`. We usually need all files under `dist`. If something we don't need, we can update the pattern to exclude them.